RLTest/__main__.py

```diff
@@ -389,12 +414,15 @@ def execute(self):
         endTime = time.time()
 
         print Colors.Bold('Test Took: %d sec' % (endTime - startTime))
-        print Colors.Bold('Total Tests Run: %d, Total Tests Failed: %d, Total Tests Passed: %d' % (done, len(self.testsFailed), done - len(self.testsFailed)))
-        if len(self.testsFailed) > 0:
-            print Colors.Bold('Faild Tests Summery:')
-            for testFaild in self.testsFailed:
-                print '\t' + Colors.Bold(testFaild.testNamePrintable)
-                testFaild.printFailuresSummery('\t\t')
+        print Colors.Bold('Total Tests Run: %d, Total Tests Failed: %d, Total Tests Passed: %d' % (done, self.getTotalFailureCount(), done - self.getTotalFailureCount()))
+        if self.testsFailed:
+            print Colors.Bold('Failed Tests Summary:')
+            for group, failures in self.testsFailed:
+                print '\t' + Colors.Bold(group)
+                if not failures:
+                    print '\t\t(Exception raised during test execution. See logs)'
+                for failure in failures:
+                    print '\t\t' + failure
```

Comment:
you do not print the failure summery here, why?

Reply:
I do, but instead of assuming the failure is a Env object, it is now a list of strings. It looks the same formatted.